engine/engine-script-library

```diff
@@ -351,7 +351,11 @@ function setup_core_env() {
 
 function get_data() {
     # Get files required to run benchmark and tools
-    cs_files_list="$cs_type-files-list"
+    if [ $cs_type == "client" -o $cs_type == "server" -o $cs_type == "profiler" ]; then
+        cs_files_list="$cs_type-$cs_id-files-list"
+    else # worker and master do not get id-specific files-list, at least not yet
+        cs_files_list="$cs_type-files-list"
+    fi
```

Comment:
This seems promising for what I want to do with being able to import multiple files.